pyramid_zipkin/request_helper.py

```diff
@@ -169,14 +179,15 @@ def get_binary_annotations(
     """
 
     annotations = {
-        'http.request.method': request.method,
-        'network.protocol.version': request.http_version,
-        'url.path': request.path,
-        'server.address': request.server_name,
-        'server.port': str(request.server_port),
-        'url.scheme': request.scheme,
-        'http.uri': request.path,
-        'http.uri.qs': request.path_qs,
+        'http.request.method': safe_get_property(request, 'method', 'unknown'),
```

Comment:
getattr(obj, prop, default) provides a way to specify a default value if the attribute isn't found, hence avoiding the need for exception handling?